invoices/invoiceregistry_test.go

```diff
@@ -3,17 +3,21 @@ package invoices
 import (
```

Comment:
There's quiet a bit of refactor in #3694 to move all test data generation to a separate compulation unit. I'd suggest building on that. Not sure how to do this properly without merging that first though.

Reply:
It is mostly rename/move it seems. I think it can be done independently?

Reply:
I'm fine merging this if this PR is merged to master earlier. Optionally we could somehow make it part of this PR too (it's in commit: f663b33).